Enterprise-CMCS/macpro-onemac
# OY2 23552: Latest Package Activity - Package Details Page + Dashboard

Story: https://qmacbis.atlassian.net/browse/OY2-23552
Endpoint: See github-actions bot comment

### Details
Knowing the last OneMAC Action taken is something the users want to see.   It is called the "Last Package Activity"

### Changes
- added the lastActivityTimestamp attribute to the Package Item
- included it in the getMyPackages projection
- set the value in the package builder according to the story AC
- added the column to the dashboard
- added the field to the details pages in the assigned spots
- added the submission ID field to the details pages, where requested.

### Test Plan
1. Login as a CMS user
2. Verify the Last Activity Date column matches the story AC
3. Verify the package details pages for the packages match the story AC for each package type
4. Verify the "Export to csv" button includes the last activity date and it is in a readable date format. (not in AC)
5. Logout and login as state user
6. create a new submission
7. verify the last activity date matches the initial submission date for the new submission
8. take a package action that is one of the ones listed for updating the dates
9. verify the date got updated per the story ACs.
